sled-agent/src/rack_setup/service.rs

```diff
@@ -1069,6 +1093,7 @@ impl ServiceInner {
             Ledger::<RssCompleteMarker>::new(&self.log, marker_paths.clone())
                 .await;
 
+        rss_step.update(RssStep::CheckPrevious);
```

Comment:
Bikeshed - I think I'd skip this step. The time between here and the next step (whether it's `LoadExistingPlan` or `WaitingForPeers`) should be basically zero.